File: modul2/Python/PRAK204_2310817210017_MUHAMMADNURHUDABAIHAQI.py
jari_jari, tinggi, = map(float, input().split())
volume = 22/7 *jari_jari*jari_jari*tinggi
luas = 22/7 *jari_jari*(jari_jari+tinggi)
keliling = 2*(22/7)*jari_jari
print(f"Volume = {volume:.2f}")
print(f"Luas = {luas:.2f}")
print(f"Keliling = {keliling:.2f}")

File: modul2/C/PRAK204_2310817210017_MUHAMMADNURHUDABAIHAQI.c
#include <stdio.h>
int main() 
{
    float jari_jari, tinggi;
    printf("");
    scanf("%f", &jari_jari);
    printf("");
    scanf("%f", &tinggi);
    float volume = (3.1428571428571 *jari_jari*jari_jari*tinggi);
    printf("Volume = %.2f \n", volume);
    float luas = (2*3.1428571428571*jari_jari*(jari_jari+tinggi));
    printf("Luas = %.2f \n", luas);
    float keliling = (2*3.1428571428571*jari_jari);
    printf("Keliling= %.2f \n", keliling);

    return 0;
    
} 